packages/dropzone/stories/dropzone.stories.ts

```diff
@@ -44,48 +50,140 @@ type StoryArgs = {
 export const Default = (args: StoryArgs): TemplateResult => {
     return html`
         <sp-dropzone id="dropzone" tabindex="0" ?dragged=${args.isDragged}>
-            <sp-illustrated-message heading="Drag and Drop Your File" cta>
+            <sp-illustrated-message heading="Drag and Drop Your File">
                 ${illustration}
-                <div slot="description">
-                    <label for="file-input">
-                        <sp-link>Select a File</sp-link>
-                        from your computer
-                    </label>
-                    <input type="file" id="file-input" style="display: none" />
-                </div>
-                <div slot="description">
-                    or
-                    <sp-link href="http://stock.adobe.com" target="blank">
-                        Search Adobe Stock
-                    </sp-link>
-                </div>
             </sp-illustrated-message>
+            <div>
+                <label for="file-input">
+                    <sp-link
+                        href="javascript:;"
+                        onclick="this.parentElement.nextElementSibling.click()"
+                    >
+                        Select a File
+                    </sp-link>
+                    from your computer
+                </label>
+                <input type="file" id="file-input" style="display: none" />
+            </div>
+            <div>
+                or
+                <sp-link href="http://stock.adobe.com" target="blank">
+                    Search Adobe Stock
+                </sp-link>
+            </div>
         </sp-dropzone>
     `;
 };
 
 export const Dragged = (args: StoryArgs): TemplateResult => {
     return html`
         <sp-dropzone id="dropzone" tabindex="0" ?dragged=${args.isDragged}>
-            <sp-illustrated-message heading="Drag and Drop Your File" cta>
+            <sp-illustrated-message heading="Drag and Drop Your File">
                 ${illustration}
-                <div slot="description">
-                    <label for="file-input">
-                        <sp-link>Select a File</sp-link>
-                        from your computer
-                    </label>
-                    <input type="file" id="file-input" style="display: none" />
-                </div>
-                <div slot="description">
-                    or
-                    <sp-link href="http://stock.adobe.com" target="blank">
-                        Search Adobe Stock
-                    </sp-link>
-                </div>
             </sp-illustrated-message>
+            <div>
```

Comment:
I think we should keep the `slot="description"` in the `div`, cause it's probably used as a placeholder. Unless it was causing problems?

Reply:
The description is needed if we nest the links inside the illustrated message component, but the SWC doc examples (https://opensource.adobe.com/spectrum-web-components/components/dropzone/#example, https://studio.webcomponents.dev/edit/collection/fO75441E1Q5ZlI0e9pgq/yoQJtdPFBCH1UG3Qsicc/src/index.ts?p=stories) show the links living in parallel to the illustrated message so I prefer to align to be consistent which would require removing the description slots from the divs